packages/flutter_test/lib/src/binding.dart

```diff
@@ -1109,7 +1109,6 @@ class LiveTestWidgetsFlutterBinding extends TestWidgetsFlutterBinding {
   @override
   void initInstances() {
     super.initInstances();
```

Comment:
I think the `flutter analyze` step fails because this method is not necessary anymore. Could you remove it?